src/data/getstarted.json

```diff
@@ -0,0 +1,252 @@
+[
```

Comment:
We should get this from the learning collection in the CMS

Reply:
I thought it was a static page.

Reply:
Oh right we did talk about static. Let's look in the morning. 
We may not have further designs yet.